package.json

```diff
@@ -11,10 +11,12 @@
   "author": "Zooniverse <contact@zooniverse.org> (https://www.zooniverse.org/)",
   "license": "Apache-2.0",
   "dependencies": {
+    "@zooniverse/panoptes-js": "0.0.1",
```

Comment:
I don't think this version is ready for use yet.

Reply:
👌